modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionTypeGlobalJS/build.gradle

```diff
@@ -0,0 +1,36 @@
+task runGradleTest
+
+runGradleTest {
+	dependsOn ":global-js-client-extension:build"
+
+	doLast {
+		Project clientExtensionProject = project ":global-js-client-extension"
+
+		File clientExtensionBuildDir = new File(clientExtensionProject.buildDir, "liferay-client-extension-build")
+
+		File clientExtensionConfigJSONFile = new File(clientExtensionBuildDir, "global-js-client-extension.client-extension-config.json")
+
+		assert clientExtensionConfigJSONFile.exists()
+
+		String file = _readFileAsString(clientExtensionConfigJSONFile.newInputStream())
+
+		assert file.contains("scriptElementAttributes=defer=\\\"true\\\"\\ndeferTrue=\\\"true\\\"\\ndeferStrTrue=\\\"true\\\"\\ndeferFalse=\\\"false\\\"\\ndeferStrFalse=\\\"false\\\"\\ndata-bool-true=\\\"true\\\"\\ndata-test=\\\"false\\\"\\ndata-bool=\\\"false\\\"\\ndata-empty=\\\"\\\"\\nnomodule=\\\"false\\\"\\nnulll=\\\"true\\\"")
+	}
```

Comment:
@thiago-buarqque because this is groovy, you can just do `clientExtensionConfigJSONFile.text.contains(...)` instead of reading from an input stream.